privacy-assessments/auto-expanding-details-privacy.md

```diff
@@ -0,0 +1,95 @@
+# Auto-expanding `<details>` security and privacy self-review questionnaire
+
+## Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This feature will open details elements in response to find-in-page and ScrollToTextFragment. When a details element is opened, the page can observe it via the `toggle` event which is fired and the `open` attribute. It is necessary that we continue to fire this event and add the attribute when the details element is opened because the page may be relying on them to reconcile other state in the page or in script with the open state of the details element.
+
+### 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes.
+
+### 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+If you consider find-in-page queries or ScrollToTextFragment queries persinal information or PII, it is dealt with the same way "sensitive information" is in the next section.
+
+No new information stored on the device is exposed by this feature.
+
+### 2.4. How does this specification deal with sensitive information?
+
+The search queries present in the ScrollToTextFragment and find-in-page features may be sensitive information, and are considered in the "Privacy Considerations" section.
+
+### 2.5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+This feature does not store any new data in the browser.
+
+### 2.6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+No information from the underlying platform is exposed by this feature.
+
+### 2.7. Does this specification allow an origin access to sensors on a user's device
+
+No.
+
+### 2.8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+This feature does not expose any information about an origin's state and does not enable sending or receving of any data to or from a network connection.
+
+### 2.9. Does this specification enable new script execution/loading mechanisms?
+
+No.
+
+### 2.10. Does this specification allow an origin to access other devices?
+
+No.
+
+### 2.11. Does this specification allow an origin some measure of control over a user agent's native UI?
+
+No.
+
+### 2.12. What temporary identifiers might this specification create or expose to the web?
+
+None.
+
+### 2.13. How does this specification distinguish between behavior in first-party and third-party contexts?
+
+If a match is found in a cross-origin iframe and a details element exists in a parent frame, this feature will not be used and the details element will not be automatically expanded.
```

Comment:
Is this our intent? I thought we talked about making this work

Reply:
My bad, I changed this to say that it will work for find-in-page.
Based on my experimenting, it appears that ScrollToTextFragment doesn't work on cross origin iframes.